config_draft.properties

```diff
@@ -0,0 +1,7 @@
+[connection]
+remote_device = your device address here
```

Comment:
May let's rename to `ip-address` or something like that. It has to be self-explained.